api/src/api/housingRouter.ts

```diff
@@ -8,7 +8,7 @@ export const housingRouterFactory = (housingPropertyInterest: HousingPropertyInt
   router.get("/housing/:address", async (req, res) => {
     const address = req.params.address;
     housingPropertyInterest(address)
-      .then(async (propertyInterestDetails: HousingPropertyInterestDetails[]) => {
+      .then(async (propertyInterestDetails?: HousingPropertyInterestDetails) => {
```

Comment:
dust, why is the result of the function call undefined? aren't we expecting something back? Should it hit the catch block if it's undefined?